simple_qa_rnn/relation_prediction/hyperparameter_tuning.py

```diff
@@ -10,8 +10,8 @@
     dropout = uniform(0.5, 0.6)
     clip = uniform(0.6, 0.7)
 
-    command = "python train.py --cuda --device 1 --dev_every 500 --log_every 250 --batch_size 128 " \
-                "--epochs {} --lr {} --d_hidden {} --n_layers {} --dropout_prob {} --clip {} >> " \
+    command = "python train.py --cuda --gpu 1 --dev_every 500 --log_every 250 --batch_size 32 " \
```

Comment:
This code is open-source should make as few assumptions about the running environment as possible to make it easy for someone else to reproduce. Ideally someone can just pull the code and run but it might not work because they might not have 2 GPU cores.